### PR TITLE
Added service object documentation. Standardized object preambles.

### PR DESCRIPTION
**Pull request by mzhaase**
The preamble was sometimes different. Now they all follow the same format. Some minor corrections to formatting. Added service object documentation.

### CODE REVIEW

**Comment by arioch** [comment]
Excellent stuff. :+1: 
Before I can merge could you rebase those commits into a single one? 
(`git rebase -i HEAD~3; fixup last two commits; git push $remote +develop`...)

File: README.md [hunk 1]
```diff
@@ -1020,11 +1024,43 @@ icinga2::object::scheduleddowntime {'some-downtime':
...
+Example usage:
+
+<pre>
+icinga2::object::service { 'example_service':
+$template_to_import = 'generic-service',
+$display_name = 'Example Service',
+$host_name = $::fqdn,
+$groups = ['example-servers', 'linux-servers'],
+$vars = {
+  var1 = 'somevalue',
```
Comment (lazyfrosch) [inline README.md:1038]: Can you fix indention here?

Comment (lazyfrosch) [inline README.md:1038]: And this is not declaration syntax :)

**Comment by arioch** [comment]
Merged and fixed indentation and syntax errors too.
Thanks!

**Pull request closed without merging**